src/profile.command.ts

```diff
@@ -43,6 +43,24 @@ export abstract class ProfileCommand extends Command {
                 ' --network network --url http://localhost:3000');
         }
     }
+
+    public setCurProfile(options: ProfileOptions): Profile {
+        const profileName = options.profile;
+        try {
+            return this.profileService.changeCurProfile(profileName);
+        } catch (err) {
+            throw new ExpectedError(options.profile ? 'Set current profile fail' :
```

Comment:
We could try to give some more guidance to the users. For example "The profile was not saved because ... <reason>." If possible, mention how to set a valid profile name.